ui/hooks/useAccountTotalCrossChainFiatBalance.ts

```diff
@@ -56,7 +57,12 @@ export const useAccountTotalCrossChainFiatBalance = (
     () =>
       formattedTokensWithBalancesPerChain.map((singleChainTokenBalances) => {
         const { tokensWithBalances } = singleChainTokenBalances;
+        // Attempt to use known currency symbols in map
+        // Otherwise fallback to user defined currency
         const matchedChainSymbol =
+          CHAIN_ID_TO_CURRENCY_SYMBOL_MAP[
+            singleChainTokenBalances.chainId as keyof typeof CHAIN_ID_TO_CURRENCY_SYMBOL_MAP
+          ] ??
```

Comment:
Try to use original native currency symbol before the user modifiable network ticker. Ensures that the aggregated balance (if shown in USD) does not point to an incorrect conversion rate.